packages/client/src/util/inclineClient.ts

```diff
@@ -21,20 +22,29 @@ export async function createInlineClient(
   let stateDB
   let metaDB
   if (memoryDB) {
-    chainDB = new MemoryLevel<string | Uint8Array, string | Uint8Array>()
-    stateDB = new MemoryLevel<string | Uint8Array, string | Uint8Array>()
-    metaDB = new MemoryLevel<string | Uint8Array, string | Uint8Array>()
+    chainDB = new MemoryLevel<
+      string | Uint8Array,
+      string | Uint8Array
+    >() as unknown as AbstractLevel<string | Uint8Array, string | Uint8Array, string | Uint8Array>
```

Comment:
Can we add similar notes as to why we can safely do this, as you added in the typecasting of Level in the other file?

Reply:
Fixed